frontend-cert/css-projects/css-colors/user-stories.md

```diff
@@ -0,0 +1,25 @@
+1. You should set the background color for `body` to `#f4f4f4`.
+
+2. You should have a `div` with a class of `color-grid`.
+
+3. You should have five `div` elements within `div` created above.
+
+4. The first `div` should have these classes: `color-box color1`. 
```

Comment:
This is looking good @ihechikara 👍 

Here's some slight adjustments to the user stories that make it a little more specific for some things, but also more open-ended for some of the colors:

You should set the background color for `body` to `#f4f4f4`.
You should have a `div` with a class of `color-grid` to hold all your color elements.
You should have five `div` elements within the `.color-grid` element.
The five `div` elements should each have a class of `color-box` and `color#`, where `#` is the number of the order of that `div`. For example, `color1` for the first `div`, and `color2` for the second, etc.
All of your `.color-box` elements should have a set `width` and `height` so they are visible on the page.
The `.color1` element should have a `background-color` that uses a predefined (word) color value.
The `.color2` element should have a `background-color` that uses a hexadecimal color value.
The `.color3` element should have a `background-color` that uses an RGB color value.
The `.color4` element should have a `background-color` that uses a HSL color value.
The `.color5` element should have a `background-color` set.

I changed it so that one of each type of color values is required, but not defined - so users can pick the colors, but they have to use all types.

Also note that if we change the user stories to be more open-ended like this, the prototype won't fulfill them anymore.

Let me know what you think @ihechikara.

Reply:
Thanks for the suggestions, @moT01! I've updated the PR - let me know what you think